[Prometheus] Update default dashboard, update sample_event files (#4275)
elastic/integrations: packages/prometheus/_dev/build/docs/README.md

# Prometheus Integration

This integration periodically fetches metrics from [Prometheus](https://prometheus.io/) metrics endpoints.
This integration can collect metrics from Prometheus Exporters, receive metrics from Prometheus server using Remote Write

> and execute specific Prometheus queries against Promethes Query API.

Reviewer: ```suggestion
or execute specific Prometheus queries against Promethes Query API.
```

The fields reported are:

{{fields "query"}}

## Dashboard

Prometheus integration is shipped including default overview dashboard.

> Default dashboard works only for `remote_write` datastream and `collector` darastream, if metrics are scraped from the Prometheus server metrics endpoint.

Reviewer: ```suggestion
Default dashboard tested with `remote_write` datastream and `collector` darastream, if metrics are scraped from the Prometheus server metrics endpoint.
```

Reviewer: Dont know if we want to write something like:
Provided dashboards can not guarantee coverage in case of Prometheus Query configuration scenarios

Author: This dashboard will not work with query, as metrics will be stored under `prometheus.query.*`, that is why I wrote `Default dashboard works only for ...`
